caom2utils/caom2utils/caom2blueprint.py

```diff
@@ -3161,6 +3187,14 @@ def _two_param_constructor(self, lookup1, lookup2, index, to_type, ctor):
             new_object = ctor(param1, param2)
         return new_object
 
+    # TODO - is this the right implementation?
+    def ignore_chunks(self, artifact, index=0):
```

Comment:
It's `ignore_chunks` because the original Java implementation configuration used the `Chunk = {ignore}` in the override file, but I will rename it to `add_parts`, and invert the return value.